README.md

```diff
@@ -4,51 +4,62 @@
-- Highly ergonomic for repl-driven development.
+Snitch is how I debug and understand data flowing through my system. I use it as my sole debugger (in rare cases I reach for print statements).
 
-# What people have to say ❤️ 
+It's not a replacement for a full-fledged debugger, but it's pretty close and will be useful in 90% of the cases. However, unlike a debugger Snitch runs while your application us running, which is both better and worse than a traditional debugger. Better because a debugger freezes your application while you inspect the state. Better because the Clojure REPL will let you modify the code you are debugging, while it is running. Most debugger do not let you do that. Worse because Snitch will only show you the latest value from a series of invocation, where a debugger will let you step to the invocation you are interested in. (BTW. There are ways to collect state from all invocations, even if Snitch doesn't help with it.)
+
+# Features
+- `defn*`, `*let`, `*fn`, and `defmethod*` macros automating inline `def`s for function arguments and bindings
+- Gives access to the return value of called functions
+- Lets you modify and then rerun the function with the snitched invokation arguments
+- Support for Clojurescript.
+- Editor agnostic (use it along with [CIDER](https://cider.mx), [Conjure](https://conjure.oli.me.uk/), [Calva](https://calva.io), [Cursive](https://cursive-ide.com/), or any REPL enhanced [Clojure editor](https://clojure.org/guides/editors!).
+
+# What people have to say ❤️
 > Very handy with those variants of the regular macros. Just add a `*` and you are inline def-ing the args! - Peter Strömberg (co-author of Calva)
 
 # Talk
-I gave a [talk](https://www.youtube.com/watch?v=WqilQulsJQc) about snitch at clojure Asia you can watch it 
+I gave a [talk](https://www.youtube.com/watch?v=WqilQulsJQc) about snitch at clojure Asia you can watch it
 here https://www.youtube.com/watch?v=WqilQulsJQc.
 
 # Usage
 ## Installation
-I recommend adding snitch to your `~/.lein/profiles.clj`.
+I recommend adding snitch to your `~/.lein/profiles.clj` or `deps.edn`.
 An example file would be
 ```clojure
-; profiles.clj
+; Leiningen profiles.clj
 {:user {:dependencies [[org.clojars.abhinav/snitch "0.1.16"]]}}
 
 {:dev {:dependencies [[org.clojars.abhinav/snitch "0.1.16"]]}}
 ```
-If you add it to your project's dev dependencies, you could add this line to your dev/user.clj
-```
+
+Once your repl is running you can evaluate:
+
+```clojure
 (require '[snitch.core :refer [defn* defmethod* *fn *let]])
 ```
-requiring these macros once, will intern these macros inside clojure.core & cljs.core, so you don't have to import them in every namespace.
+
+in the Clojure REPL and the macros will be interned inside clojure.core & cljs.core, so you don't have to import them in every namespace. It will also make the macros available for your ClojureScript namespaces if your ClojureScript REPL is cloned/spawned off of the Clojure REPL.
```

Comment:
> It will also make the macros available for your ClojureScript namespaces if your ClojureScript REPL is cloned/spawned off of the Clojure REPL.

wait, really? that is so cool, I didn't know this!

Reply:
It's indeed very cool. I add the dependency and do the require in one evaluation and then Snitch is there wherever I need it. And I need it everywhere!

Reply:
I didn't even know that was possible so thanks for mentioning it 🤯